### PR TITLE
Updated support platform list

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Fixes https://github.com/eclipse-openj9/openj9-docs/issues/1059

Supported platforms list updated

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the topics. Please check and confirm. Thanks!

File: docs/version0.37.md [hunk 1]
```diff
@@ -33,11 +33,9 @@ The following new features and notable changes since version 0.36.x are included
 
 ### Binaries and supported environments
 
-Eclipse OpenJ9&trade; release 0.36.0 supports OpenJDK 8 and 17.
+Eclipse OpenJ9&trade; release 0.37.0 supports OpenJDK 19.
 
-Release 0.36.1 supports OpenJDK 11.
-
-Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release. OpenJ9 will not be tested with OpenJDK 11 on CentOS 6.10 after the 0.36.1 release.
```
Comment (pshipton) [inline docs/version0.37.md:40]: I don't think this should be removed so quickly.

**Comment by pshipton** [comment]
Ubuntu 18.04 should be removed, as per https://github.com/eclipse-openj9/openj9-docs/issues/1059

File: docs/version0.37.md [hunk 2]
```diff
@@ -33,11 +33,9 @@ The following new features and notable changes since version 0.36.x are included
...
+RHEL 8.4 is now supported but RHEL 8.2 is out of support.
```
Comment (pshipton) [inline docs/version0.37.md:38]: RHEL 8.4 was always supported, we only list the minimum levels. 8.4 is the new minimum level.

**Comment by Sreekala-Gopakumar** [comment]
> Ubuntu 18.04 should be removed, as per #1059

As per https://github.com/eclipse-openj9/openj9-docs/issues/1045, this was to be removed for Release 0.38.0 (April), that is why I didn't delete for Release 0.37.0 (March). Do you want me to delete it for 0.37.0 release itself?

File: docs/version0.37.md (hunk 1 above)
Comment (Sreekala-Gopakumar) [reply docs/version0.37.md:40]: So, should I replace 0.36.1 with 0.37.0 in these sentences?

**Comment by pshipton** [comment]
> Do you want me to delete it for 0.37.0 release itself?

Yes pls. It will out of support so soon after 0.37 it's not worth keeping it listed.

File: docs/version0.37.md (hunk 1 above)
Comment (pshipton) [reply docs/version0.37.md:40]: I'm only referring to the CentOS 6.10 statements. Using 0.36.1 is still applicable, 0.37 is after 0.36.1, but you could write "for the 0.37 release." and keep updating it if you like.

**Review by pshipton — APPROVED** [review]

**Pull request merged**